scripts/mkcloud

```diff
@@ -1202,6 +1222,11 @@ Optional
         Default value will be applied from Keystone barclamp proposal.
     want_cd='' | 1  (default='')
         Set up a cloud based on the continuous delivery channel.
+    want_ceph=1 (default=1)
+        Setup an internal ceph cluster using the ceph barclamp.
+    want_external_ceph=5 (default=0)
```

Comment:
Can we use the "clusterconfig" for this as well, so we don't have to deal with two different paradigms of specifying a cluster?

Reply:
oh maybe, I not sure what that is.. but from a quick look, that kinda makes sense :)

Reply:
Please don't misuse the clusterconfig for this. :)
Clusterconfig is made to define pacemaker clusters.
The definition of nodes and their purpose is something that I am currently refactoring. I alread started this, but other priorities are currently blocking it. But when its done, then you can define with any flexibility you want, how many nodes of which kind you want to create and what should be deployed on them.

I don't have a spontaneous workaround for the time in between. Maybe I need to think about it a bit more.